application/models/Participant.php

```diff
@@ -2124,7 +2124,8 @@ public function insertParticipantCSV($data)
             'language' => $data['language'],
             'blacklisted' => $data['blacklisted'],
             'created_by' => $data['owner_uid'],
-            'owner_uid' => $data['owner_uid']);
+            'owner_uid' => $data['owner_uid'],
+            'created' => date('Y-m-d H:i:s', time()));
```

Comment:
Need time shift ? No ? (`timeadjust`)
https://github.com/LimeSurvey/LimeSurvey/blob/1daf9d4347b9a102a26aa8ba67e70a20fd98438a/application/helpers/common_helper.php#L2491

Reply:
Will fix

Reply:
OK, so after some review I got the following.
- When a token is created, `created ` is assigned `NOW()` from DB.
- When a token is copied, `created ` is assigned `date('Y-m-d H:i:s', time()));`
https://github.com/LimeSurvey/LimeSurvey/blob/1d87c0e7c3fa972b32b16a2ef0ea052376d81a2e/application/models/Participant.php#L2030

So, seems dateShift is not used here at the Participants Model.
Maybe we should, but we could open a new ticket for that as to review it specificlly and handle it easier.
What do you think?

Reply:
> So, seems dateShift is not used here at the Participants Model.

Seems to be an issue :) 

>  Maybe we should, but we could open a new ticket for that as to review it specificlly and handle it easier.

Yes :+1: (but i'm on holiday on August …)

Reply:
https://bugs.limesurvey.org/view.php?id=16568

Happy hollidays!